evap/staff/templates/staff_user_list.html

```diff
@@ -0,0 +1,105 @@
+                        <td>{{ user.full_name }}</td>
+                        <td>{% if user.email %}{{ user.email }}{% endif %}</td>
+                        <td>{% include 'staff_user_badges.html' with user=user %}</td>
+                        <td>
+                            <a href="{% url 'staff:user_edit' user.id request.path %}" class="btn btn-secondary btn-sm"><span class="fas fa-pencil-alt"></span></a>
+                            {% if user.can_be_deleted_by_manager %}
+                                <button type="button" class="btn btn-danger btn-sm" onclick="deleteModalShow({{ user.id }}, '{{ user.full_name|escapejs }}');">
+                                    <span class="fas fa-trash"></span
+                                ></button>
+                            {% else %}
+                                <button type="button" disabled class="btn btn-sm btn-danger" data-toggle="tooltip" data-placement="left"
+                                    title="{% blocktrans %}This user contributes to an evaluation, participates in an evaluation whose participations 
+                                    haven't been archived yet or has special rights and as such cannot be deleted.{% endblocktrans %}">
+                                    <span class="fas fa-trash"></span>
+                                </button>
+                            {% endif %}
+                        </td>
+                    </tr>
+                {% endfor %}
+                </tbody>
+            </table>
+        </div>
+    </div>
+{% endblock %}
+
+{% block modals %}
+    {{ block.super }}
+    {% trans 'Delete user' as title %}
+    {% trans 'Do you really want to delete the user <strong id="data-label"></strong>?<br/>This person will also be removed from every other user having this person as a delegated or CC-user.' as question %}
+    {% trans 'Delete user' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='deleteModal' title=title question=question action_text=action_text btn_type='danger' %}
+    <script type="text/javascript">
+        function deleteModalAction(dataId) {
+            $.ajax({
+                type: "POST",
+                url: "{% url 'staff:user_delete' %}",
+                data: {"user_id": dataId},
+                success: function(){ $('#user-row-'+dataId).hide('slow', function(){ $('#user-row-'+dataId).remove(); }); },
+                error: function(){ window.alert("{% trans 'The server is not responding.' %}"); }
+            });
+        };
+    </script>
+{% endblock %}
+
+{% block additional_javascript %}
+    <script type="module">
+        import {TableGrid} from "{% static 'js/datagrid.js' %}";
+        new TableGrid().init({
+            storageKey: "user-index-data-grid",
+            table: $(".user-table"),
+            searchInput: $("input[name=search]"),
+            resetSearch: $("[data-reset=search]"),
+        });
+    </script>
+{% endblock %}
```

Comment:
The file is missing a newline character at the end of the last line. There is a preference for this in VS:Code somewhere.